spec/rubocop/cop/rspec/capybara/visibility_matcher_spec.rb

```diff
@@ -35,10 +35,24 @@
                                                 ^^^^^^^^^^^^^^ Use `:all` or `:hidden` instead of `false`.
       expect(page).to have_unchecked_field('cat', visible: false)
                                                   ^^^^^^^^^^^^^^ Use `:all` or `:hidden` instead of `false`.
-      expect(page).to have_text('My homepage', visible: false)
-                                               ^^^^^^^^^^^^^^ Use `:all` or `:hidden` instead of `false`.
-      expect(page).to have_content('Success', visible: false)
+      expect(page).to have_no_css('.profile', visible: false)
```

Comment:
I start to wonder if we shouldn't generate the test in a loop over the said selectors

Reply:
It is indeed a bit long, so I cut out the negative matcher separately. WDYT?

Reply:
Works for me